CHANGELOG.md

```diff
@@ -5,16 +5,23 @@ Full documentation for Composable Kernel is not yet available.
 ## (Unreleased) CK for ROCm 6.0.0
 
 ### Fixed
+ - Fixed a hazard with using inline v_dot (#808).
+ - Fixed a bugs in grouped Convolution backward data with not padded K (#848 #876).
 
 ### Optimizations
 
 ### Added
 - Added image to column (#867) and column to image kernels (#930).
+- Added support for 3D grouped Convolution forward on Navi3x (#935).
```

Comment:
Navi3x is an internal term. Navi3 GPU family? @LisaDelaney we should standardize this. Please add to spell check if possible I think a tm may be needed as well.

Reply:
Looks like it should be AMD RDNA 3? 

> Navi3x is an internal term. Navi3 GPU family? @LisaDelaney we should standardize this. Please add to spell check if possible I think a tm may be needed as well.

Looks like maybe AMD RDNA 3?